lib/rack/attack.rb

```diff
@@ -32,8 +32,14 @@ class IncompatibleStoreError < Error; end
     autoload :Fail2Ban,             'rack/attack/fail2ban'
     autoload :Allow2Ban,            'rack/attack/allow2ban'
 
+    DEFAULT_ALLOWED_ERRORS = %w[Dalli::DalliError Redis::BaseError].freeze
+
     class << self
-      attr_accessor :enabled, :notifier, :throttle_discriminator_normalizer
+      attr_accessor :enabled,
+                    :notifier,
+                    :throttle_discriminator_normalizer,
+                    :allowed_errors
```

Comment:
I see this as a new configuration, so I'd move it to `lib/rack/attack/configuration.rb`. See `throttled_responder` as an example. WDYT? Do you see any reason why to have it here instead?

Reply:
@santib done.